src/ralph/backends/database/base.py

```diff
@@ -57,38 +55,20 @@ class AgentParameters:
     account__home_page: Optional[str] = None
```

Comment:
We could take advantage of this change to create an `account` pydantic model. It could be done in another PR.
WDYT?

Reply:
Yup, in another PR indeed!